pkg/handler/processor/process/process.go

```diff
@@ -91,8 +93,8 @@ func processDocument(i *processor.Document) ([]*processor.Document, error) {
 	return ds, nil
 }
 
-func preProcessDocument(i *processor.Document) error {
-	docType, format, err := guesser.GuessDocument(i)
+func preProcessDocument(ctx context.Context, i *processor.Document) error {
+	docType, format, err := guesser.GuessDocument(logging.WithLogger(ctx), i)
```

Comment:
WithLogger shouldn’t need to be done here, it should already be initialized from the context passed in.

Reply:
Addressed ✅